plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -17,14 +17,21 @@ class Direction(Enum):
     maximize = 2
 
 
+class Sampler(Enum):
+    TPESampler = 1
+    RandomSampler = 2
+    CmaEsSampler = 3
```

Comment:
I recommend lowercase as users will need to type to override the samplers from the command line,

Reply:
```suggestion
    tpe = 1
    random = 2
    cmaes = 3
```